nilearn/masking.py

```diff
@@ -566,15 +567,20 @@ def compute_gray_matter_mask(target_img, threshold=.5,
     mask : nibabel.Nifti1Image
         The brain mask (3D image)
     """
+
     return compute_brain_mask(target_img=target_img, threshold=threshold,
                               connected=connected, opening=opening,
-                              memory=memory, verbose=verbose)
+                              memory=memory, verbose=verbose,
+                              mask_type='whole-brain',
+                              resolution=2)
 
 
-def compute_brain_mask(target_img, threshold=.5, connected=True,
-                       opening=2, memory=None, verbose=0):
-    """Compute the whole-brain mask. This mask is calculated through the
-    resampling of the MNI152 template mask onto the target image.
+def compute_brain_mask(target_img, threshold=.5, connected=True, opening=2,
+                       memory=None, verbose=0, mask_type='whole-brain',
+                       **kwarg):
```

Comment:
I'm not sure to understand why resolution was added like this. Why not simply `resolution=2`?
It would be easier to add a deprecation warning for this value being updated to 1mm in release 0.9.

Reply:
:+1:  (as discussed above this parameter should also appear in `load_mni152_template`)

I think a `**` parameter was used here because I left a confusing comment [before](https://github.com/nilearn/nilearn/pull/2738#discussion_r628484107), sorry about that -- I meant add `resolution` as a parameter with a default value, eg `load_mni152_template(resolution=2)`